core/src/main/java/org/fao/geonet/kernel/security/ldap/LDAPUtils.java

```diff
@@ -120,6 +120,7 @@ protected User getUser(LDAPUser user, boolean importPrivilegesFromLdap,
             toSave = user.getUser();
         }
         toSave.getSecurity().setAuthType(LDAPConstants.LDAP_FLAG);
+        toSave.getSecurity().setPassword(User.getRandomPassword()); //Oracle doesn't allow a password as '' (interpreted as null)
```

Comment:
The Oracle comment is a little misleading.  Are you setting the random password because Oracle does not allow null?  If so then maybe set the password to the word 'null' and the Oracle comment would make more sense.
Also if the password is supposed to be null and it is being set to a random value, there is no way to know if this is a real password or a null password in case this information would be required later.  i.e. if null password then don't pass credentials.

Reply:
That password should not be used as the authentication is delegated to LDAP, but agree a fixed value can make it more clear.

Reply:
changed to "NULL"